CONTRIBUTING.md

```diff
@@ -224,12 +198,11 @@ If you are a project maintainer, you can build and release a new version of
   - [ ] Update the version number and dex count badge by running `make VERSION=[number] bump`
   - [ ] Inspect the updated CHANGELOG, README, and version files to ensure they are correct
 - Once merged:
-  - Pull the latest changes (checking out master if necessary) and build by running `./gradlew sdk:assembleRelease`
+  - Pull the latest changes (checking out master if necessary) and build by running `./gradlew assembleRelease`
   - Release to GitHub:
     - [ ] Run `git tag vX.X.X && git push origin --tags`
     - [ ] Create a release from your new tag on [GitHub Releases](https://github.com/bugsnag/bugsnag-android/releases)
-    - [ ] Upload the generated `.aar` file from `{sdk,ndk}/build/outputs/aar/bugsnag-android-*.aar`
```

Comment:
Is there some discussion about removing the artifacts from GitHub releases?

Reply:
Manual installation has already been removed from our docs - it seems reasonable to ask anyone integrating an AAR manually to download it themselves from mavenCentral/bintray.